Formula/v/vcctl.rb

```diff
@@ -0,0 +1,19 @@
+class Vcctl < Formula
+  desc "Cloud Native Batch System (Project under CNCF)"
+  homepage "https://volcano.sh"
+  url "https://github.com/volcano-sh/volcano/archive/refs/tags/v1.12.2.tar.gz"
+  sha256 "07f89c901336bf9a85c0984c255d4600852f7465de4878bb0d530543bb7dfe97"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+  depends_on "make" => :build
+
+  def install
+    system "make", "vcctl"
+    bin.install "_output/bin/vcctl"
```

Comment:
use system "go", ...

Reply:
hmmm, I guess I don't 100% follow here.. the volcano cli has an official installation [guide](https://volcano.sh/en/docs/cli/) to use `make vcctl` and not "explicitly" using `go ...`, should I still make a change to manually `go build` it?

 
<img width="981" height="466" alt="Screenshot 2025-09-13 at 12 21 39 AM" src="https://github.com/user-attachments/assets/2509eb1d-479d-4734-b97b-7a37a4e70b46" />